eng/common/scripts/GeneralReleaseNotesLogic.ps1

```diff
@@ -0,0 +1,128 @@
+. (Join-Path $PSScriptRoot common.ps1)
+
+$PATTERN_REGEX = "^\[\w+(\.(?<SectionName>\w+))?\]:\s#\s\((?<Pattern>.*)\)"
+$CsvMetaData = Get-CSVMetadata
+
+function Get-PackagesInfoFromFile ($releaseNotesLocation) 
+{
+    $releaseNotesContent = Get-Content -Path $releaseNotesLocation
+    $checkLine = $False
+    $presentPkgInfo = @()
+
+    foreach ($line in $releaseNotesContent)
+    {
+        if ($line -eq "<!--")
+        {
+            $checkLine = $True
+            continue
+        }
+        if ($line -eq "-->")
+        {
+            break
+        }
+        if ($checkLine)
+        {
+            $pkgInfo = ($line.Trim()).Split(":")
```

Comment:
You probably should move the trim to the start of this block so that the other checks also benefit from it.